wsmaster/che-core-api-devfile/src/test/java/org/eclipse/che/api/devfile/server/convert/tool/kubernetes/KubernetesToolToWorkspaceApplierTest.java

```diff
@@ -274,6 +304,10 @@ private KubernetesList toK8SList(String content) {
     return unmarshal(content, KubernetesList.class);
   }
 
+  private KubernetesList toK8SList(List<HasMetadata> list) {
```

Comment:
Could you add support of openshift template case here?

Reply:
As far as I understand resolving of objects in OpenShift template requires live OpenShift cluster.
In these test cases, I assume that K8sRecipeParser works fine and just mock its behavior https://github.com/eclipse/che/blob/4f489a8539fde7940d879a67d1582f7d8755e921/wsmaster/che-core-api-devfile/src/test/java/org/eclipse/che/api/devfile/server/convert/tool/kubernetes/KubernetesToolToWorkspaceApplierTest.java#L281-L282

Well, actuall parsing of `template`, `single object`, `list` in recipe may be covered in a separate integration test, but now I don't have clear idea how to implement it to be able to run it on CI.

Reply:
You are right, not a good case for a unit test here